companies/bancopichincha-es.json

```diff
@@ -0,0 +1,43 @@
+{
+    "slug": "bancopichincha-es",
+    "relevant-countries": [
+        "es"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "Banco Pichincha España S.A",
+    "address": "Calle Lagasca 4\n28001, Madrid\nEspaña",
+    "phone": "+34 914 36 25 94",
+    "email": "dpo@bancopichincha.es",
+    "sources": [
+        "https://www.bancopichincha.es/politica-privacidad/"
+    ],
+    "required-elements": [
+        {
+            "desc": "Nombre",
+            "type": "name",
+            "optional": false
+        },
+        {
+            "desc": "Dirección",
+            "type": "address",
+            "optional": false
+        },
+        {
+            "desc": "NIF/DNI",
+            "type": "input",
+            "optional": false
+        }
+    ],
+    "request-language": "es",
+    "custom-access-template": "https://media.bancopichincha.es/wp-content/uploads/RGPD_FORMULARIO_WEB_EJERCICIO_DERECHO_ACCESO_V01_BPE_NUEVO-LOGO.pdf",
```

Comment:
The `custom-*` values are intended for templates in https://github.com/datenanfragen/data/tree/master/templates afaik :/

But these are helpful docs, maybe add them as a comment.